File: SAM Issue 3/updated SAM files/C#/Q02cCorrect.cs
using System;
namespace Q02c
{
    class Q02c
    {
        static void Main(string[] args)
        {
            int[] attendanceList = { 800, 1499, 1600, 200 };
            int[] incomeList = { 23000, 10000, 47000, 10000 };
            int income;
            int attendance;
            for (int i = 0; i < attendanceList.Length; i++)
            {
                income = incomeList[i];
                attendance = attendanceList[i];
                Console.WriteLine("Attendance: " + attendance + " income: " + income);

                if ((attendance >= 1500) || (income >= 45000))
                    Console.WriteLine("Sufficient profit made this week");
                else if ((attendance >= 750) && (income >= 22500))
                    Console.WriteLine("income in line with attendance this week");
                else if (attendance < 500)
                    Console.WriteLine("Attendance is very low this week.  Contact the fan club.");
                else
                    Console.WriteLine("Possible accounting error.");
            }

            Console.ReadKey();
        }
    }
}

File: SAM Issue 3/updated SAM files/Java/Q02cCorrect.java
package q02c;
public class Q02c 
{
    public static void main(String[] args) 
    {
        int[] attendanceList = {800, 1499, 1600, 200};
        int[] incomeList = {23000, 10000, 47000, 10000};
        int income;
        int attendance;
        
        for (int i=0; i<attendanceList.length; i++)
        {
           income = incomeList[i];
           attendance = attendanceList[i];
           System.out.println("Attendance: " + attendance + " income: " + income);

           if ((attendance >= 1500) || (income >= 45000))
               System.out.println("Sufficient profit made this week");
           else if ((attendance >= 750) && (income >= 22500))
               System.out.println("income in line with attendance this week");
           else if (attendance < 500)
               System.out.println("Attendance is very low this week.  Contact the fan club.");
           else
               System.out.println("Possible accounting error.");
         }   
    }   
}
